Doc/library/stdtypes.rst

```diff
@@ -802,6 +802,31 @@ number, :class:`float`, or :class:`complex`::
            hash_value = -2
        return hash_value
 
+.. _typebool:
+
+Boolean Type - :class:`bool`
+============================
+
+Booleans represent truth values. The :class:`bool` type has exactly two
+constant instances ``True`` and ``False``.
+
+Other types can be cast to bool explicitly using the function :func:`bool`.
+Additionally, they are implicitly interpreted as bool in some contexts - see
+:ref:`truth`.
+
+For logical operations, use the :ref:`boolean operators <boolean>` ``and``,
+``or`` and ``not``.
+When applying the bitwise operators ``&``, ``|``, ``^`` to two booleans, they
+return a bool equivalent to the logical operations "and", "or", "xor". Still,
+for ``&`` and ``|``, the logical operators ``and`` and ``or`` are preferred.
+The use of the bitwise inversion operator ``~`` is deprecated and will raise
```

Comment:
Maybe use the `.. deprecated::` role to mention the deprecation

Reply:
done.